hazelcast/src/test/java/com/hazelcast/quorum/impl/RecentlyActiveQuorumFunctionTest.java

```diff
@@ -26,6 +30,8 @@
 import static org.junit.Assert.assertFalse;
 import static org.junit.Assert.assertTrue;
 
+@RunWith(HazelcastSerialClassRunner.class)
```

Comment:
Chosen `HazelcastSerialClassRunner` and no `ParallelTest` as in the other implementations of `AbstractQuorumFunctionTest`